README.md

```diff
@@ -30,3 +30,5 @@ I am open to translating the app to any known language. There is several ways of
 - The easiest way is [crowdin](https://crowdin.com/project/abrynosshoppinglist). You can just input the strings into a graphical user interface and I will do the rest. Please open an issue if your preferred language is not available as an option.
 - The second possibility is to open a merge request adding the translated strings in source code directly
 
+# Collaboration
+If you have Ideas or Feature Request, see the list of open Ideas with [welcomed pull requests](https://github.com/Abrynos/ShoppingList/issues?q=label%3A%22%F0%9F%91%8D+merge+request+okay%22+is%3Aclosed)
```

Comment:
Generally a nice idea, but your sentence is more in line with German orthography rather than English (lots of uppercase words). The list of issues is already linked in the checklist for a new feature-request issue (tho not filtered for "close" state) so I think this is only necessary for contributors who do not have a clear idea yet.

On another note, the whole repository uses the wording "merge request" instead of "pull request" and I'd prefer if we kept that consistant.

Feel free to use below suggestion as input or completely rewrite it in your own words.

```suggestion
If you would like to contribute but do not have a specific idea yet, feel free to look at the list of [previously suggested ideas](https://github.com/Abrynos/ShoppingList/issues?q=label%3A%22%F0%9F%91%8D+merge+request+okay%22). If you already have a specific idea but you are unable to find it in the list of previous suggestions you can either open an issue with your feature request or open a merge request directly.
```